app/controllers/user_reservations_controller.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+class UserReservationsController < ApplicationController
+  include JSONAPI::Fetching
+
+  def index
+    reservations = current_user.reservations
+
```

Comment:
What about setting a pundit policy here?  :smile:

Reply:
Will the `current_user` not do the job? 🤔 I have added this policy in the index method inside employee reservations controller though.

Reply:
Indeed it should be fine :ok_hand: